hypothesis-python/docs/reference/integrations.rst

```diff
@@ -181,7 +173,7 @@ While the observability format is agnostic to the property-based testing library
 Choices metadata
 ++++++++++++++++
 
-These additional metadata elements are included in ``metadata`` (as e.g. ``metadata["choice_nodes"]`` or ``metadata["choice_spans"]``), if and only if |OBSERVABILITY_CHOICES| is set.
+These additional metadata elements are included in ``metadata`` (as e.g. ``metadata["choice_nodes"]`` or ``metadata["choice_spans"]``), if and only if observability is configured to include choices (see |ObservabilityConfig|).
```

Comment:
We probably want a similar heading for Coverage metadata, now that it's off by default?

And also note that the format of that is unstable, since we might want to switch representations

Reply:
I'm going back and forth on whether to disable coverage by default tbh. I'm fine to leave it on for now and we can push this decision to later

> And also note that the format of that is unstable, since we might want to switch representations

done